//test exception 8

int div(int x, int y) {
    return x / y;
}

int main() {
    int s = 0;
    for (int i = 10; i >= 0; i--)
        s += div(i, i - 3);
    return s;
}
